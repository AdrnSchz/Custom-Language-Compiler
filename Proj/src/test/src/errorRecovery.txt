CEO ¡
    << Unexpected token . >>
    bro . a = 2.

    << Extra token . >>
    4 .¿ bro i = 0 . i < 5 . i++ ? ¡
        res += 2 .
    << Wrong token ¿ when expecting ! >>
    ¿
!