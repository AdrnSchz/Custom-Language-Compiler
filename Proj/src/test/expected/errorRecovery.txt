
------------------------------ Ohh, your canceled :) ------------------------------

Line 2 | Column 2 -> Unexpected token '.', expected 'name'

Line 5 | Column 1 -> Unexpected token '.', expected '¿'

Line 9 | Column 0 -> Unexpected token '¿', expected '!'

------------------------------ What a boomer ¬_¬ ------------------------------

Line 9 | Column 0 -> Variable: res, does not exist


